panc/src/test/pan/Functionality/list/list9.pan

```diff
@@ -0,0 +1,10 @@
+# put negative index greater than size
```

Comment:
There doesn't seem to be a positive index version of this test.

Reply:
indeed. i'll add one, but the behaviour is different: indices larger than the current size trigger autovivification